### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by h3rald** [h3rald/min#71] [General]
Welcome!

I wanted to kick off some discussions concerning the min programming language, trying to gather some feedback and comments from the community 😊

I know that min is not a mainstream programming language, but I am using it every day for scripting common tasks and now even building tools and applications, and I do hope you are enjoying using it as much as I am developing it!

* If you have some general feedback or questions, feel free to start a new discussion or join in on an existing one.
* If you are new to min, head over to [min-lang.org](https://min-lang.org), there's plenty of stuff to read and learn.
* If you want to see some examples of min code and compare them with other languages, head over to [Rosetta Code](http://www.rosettacode.org/wiki/Category:Min): someone actually took the time to write some pretty good code examples for common tasks.

Again, welcome and I hope you stick around!

**Comment by ghost** [comment]
A Min developer in Min developers group:  "We are *Min* developers!" -> *At the future*
